ui-components/src/helpers/filters.ts

```diff
@@ -30,7 +30,6 @@ function getComparisonForFilter(filterKey: ListingFilterKeys) {
     case ListingFilterKeys.zipcode:
       return EnumListingFilterParamsComparison["IN"]
     case ListingFilterKeys.seniorHousing:
-    case ListingFilterKeys.independentLivingHousing:
```

Comment:
I think since the eligibility questionnaire still sets it you'll still need this as well.